linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/oval/shared.xml

```diff
@@ -14,6 +14,7 @@
       <extend_definition comment="Installed OS is Ubuntu 16.04" definition_ref="installed_OS_is_ubuntu1604" />
       <extend_definition comment="Installed OS is Ubuntu 18.04" definition_ref="installed_OS_is_ubuntu1804" />
       <extend_definition comment="Installed OS is Ubuntu 20.04" definition_ref="installed_OS_is_ubuntu2004" />
+      <extend_definition comment="Installed OS is Ubuntu 22.04" definition_ref="installed_OS_is_ubuntu2204" />
```

Comment:
I don't see that Ubuntu 22.04 has been certified for FIPS 104-3 (or 104-2) on [NIST's Website](https://csrc.nist.gov/projects/cryptographic-module-validation-program/validated-modules/search?SearchMode=Basic&ModuleName=22.04&CertificateStatus=Active&ValidationYear=0) as Ubuntu 20.04 [as been](https://csrc.nist.gov/projects/cryptographic-module-validation-program/validated-modules/search?SearchMode=Basic&ModuleName=20.04&CertificateStatus=Active&ValidationYear=0). Do you have some documentation you can link to that proves otherwise? I believe we're trying to keep this list to only certified versions of operating systems.

Reply:
Ubuntu 22.04 is still going through FIPS certification, that might still not be available in the next could of months, or to be honest it will only be available in 2024. But without this change there's no way to make the rule work right? 
We could make the check in OVAL for this rule to be an OR if the system is fips certified or not. The CIS item does only mention FIPS approved algorithms, it doesn't say anything about FIPS certified platforms.
What do you think?

Reply:
I've removed the fips check and the pr to introduce 22.04 fips.